pkg/api/constants/extensions.go

```diff
@@ -0,0 +1,9 @@
+package constants
+
+// Following dist spec extension
+// https://github.com/opencontainers/distribution-spec/tree/main/extensions#extensions-api-for-distribution
+const (
+	ExtSearchPrefix      = RoutePrefix + "/_search"
```

Comment:
Move this to the end of this list with comment saying
"// zot-specific extensions"